net/http/authz/authorizer.go

```diff
@@ -155,3 +155,57 @@ func (a *Authorizer) grantsByPolicies(policies []string) ([]*Grant, error) {
 	}
 	return grants, nil
 }
+
+func (a *Authorizer) policiesByRoute(route string) ([]string, error) {
+	var (
+		n        = 0
+		policies []string
+		pat      string
+	)
+	for pattern, pol := range a.policies {
+		if !pathMatch(pattern, route) {
+			continue
+		}
+		if len(policies) == 0 || len(pattern) > n {
+			n = len(pattern)
+			policies = pol
+			pat = pattern
+		}
+	}
+	if pat == "/" {
```

Comment:
Should this be `pat == ""`? My reading of this is that it would always reject a request for `/` so the redirect to dashboard wouldn't work.